kodilla-rps/src/main/java/com/kodilla/rps/RpsRunner.java

```diff
@@ -1,8 +1,12 @@
 package com.kodilla.rps;
 
+import com.kodilla.rps.engine.Game;
+
 public class RpsRunner {
 
     public static void main(String[] args) {
+        Game game = new Game();
 
+        game.init();
```

Comment:
Jak dla mnie, to główna klasa nic nie musi wiedzieć, że jest jakiś init :) Lepiej zrobić tutaj tylko jedną linijkę: `new Game().run(); i tyle w temacie. Klasa Game sama musi się zatroszczyć o inicjalizację (delegacja - pamiętasz?)